web/src/App.js

```diff
@@ -18,8 +18,11 @@ export default function App() {
   }
 
   return (
-    <UserProvider>
-      <Routes />
-    </UserProvider>
+    <div>
+      <h1> HuskyBuddy </h1>
+      <UserProvider>
+        <Routes />
+      </UserProvider>
+    </div>
```

Comment:
Don't add any code here. Only add code in individual Screen components and then add that screen in the `Routes.js` file.